docs/runbooks/CDINoDefaultStorageClass.md

```diff
@@ -2,37 +2,39 @@
 
 ## Meaning
 
-This alert fires when there is no default (Kubernetes or KubeVirt) storage
-class, and a data volume is pending for one.
+This alert fires when there is no default (Kubernetes or virtualization) storage
+class, and a data volume is `Pending` for one.
 
-A default KubeVirt storage class has precedence over a default Kubernetes
+A default virtualization storage class has precedence over a default Kubernetes
 storage class for creating a VirtualMachine disk image.
 
 ## Impact
 
-If there is no default Kubernetes or KubeVirt storage class, a data volume that
-does not have a specified storage class remains in a "pending" state.
+If there is no default (Kubernetes or virtualization) storage class, a data
+volume that does not have a specified storage class remains in a `Pending`
+phase.
 
 ## Diagnosis
 
 1. Check for a default Kubernetes storage class by running the following
 command:
 
   ```bash
-  $ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubernetes.io/is-default-class"=="true")|.name'
+  $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubernetes\.io\/is-default-class=="true")].metadata.name}'
   ```
 
-2. Check for a default KubeVirt storage class by running the following command:
+2. Check for a default virtualization storage class by running the following
+command:
 
   ```bash
-  $ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
+  $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubevirt\.io\/is-default-virt-class=="true")].metadata.name}'
   ```
 
 ## Mitigation
 
-Create a default storage class for either Kubernetes or KubeVirt or for both.
+Create a default (Kubernetes and/or virtualization) storage class.
```

Comment:
I think here we can do something like:

`Create one or more default Kubernetes or virtualization storage classes.`

Reply:
"one or more" smells like it's ok to create for example 2 default Kuberetes and 2 virtualization storage classes, which isn't what meant here. We want to user to create either one of the defaults or both of them, but not more than that.

Reply:
Understood, thanks! Then this might be ok, but I think it can be communicated without using and/or or parentheses.

> "The construction and/or can be ambiguous and causes translation problems." (IBM Style Guide, which is one of the style guides that we follow in the docs)

Suggestion:
`Create a default storage class for Kubernetes, virtualization, or both.`